org.gnome.OfficeRunner.json

```diff
@@ -17,13 +17,23 @@
         "--socket=pulseaudio"
     ],
     "modules": [
+        {
+            "name": "gnome-common",
```

Comment:
Please split the gnome-common addition from the runtime-version bump.

Reply:
It does not build without it, do you mean two different commits?